lib/commands/alert.js

```diff
@@ -19,6 +20,25 @@ commands.getAlertText = async function () {
   }
 };
 
+helpers.nativeSetAlertText = async function (text) {
+  let alert = await this.findNativeElementOrElements('class name', 'XCUIElementTypeAlert', true);
```

Comment:
performing single element lookup should be faster in comparison to multiple elements

Reply:
BTW, I've fixed the parsing bug in https://github.com/facebook/WebDriverAgent/pull/782